<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveImageWidgetClass</class>
 <widget class="QWidget" name="SaveImageWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SaveImageWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#E0E0E0;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QGroupBox" name="MyGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,2,1,3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="CheckBox_EnableSave">
        <property name="text">
         <string>Save Original Image</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_ImgPath">
           <property name="styleSheet">
            <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btn_ChangePath">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}
QPushButton:hover
{
	background-color:#000426;
}
QPushButton:pressed
{
	background-color:rgb(65,65,65);
}</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Image Type and Location</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Image Type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_ImgType">
           <property name="styleSheet">
            <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_deleteNums">
           <property name="styleSheet">
            <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_deleteDays">
           <property name="styleSheet">
            <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_deleteMemory">
           <property name="styleSheet">
            <string notr="true">background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_deleteDays">
           <property name="text">
            <string>Days to keep Images </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_deleteNums">
           <property name="text">
            <string>Maximum number of images</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_deleteMemory">
           <property name="text">
            <string>Disk capacity lower than</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Delete Settings</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
